libass/ass_render.c

```diff
@@ -530,14 +533,14 @@ static bool quantize_transform(double m[3][3], ASS_Vector *pos,
     // D(y_out) <= (D(m_yx) * dx + D(m_yy) * dy) / z0
     //       + 2 * (D(m_zx) * dx + D(m_zy) * dy) * y_lim / z0^2.
 
-    // To estimate acceptable error in matrix coefficient
-    // set error in all other coefficients to zero and solve system
-    // D(x_out) <= ACCURACY & D(y_out) <= ACCURACY for desired D(m_kl).
-    // ACCURACY here is some part of total error, i. e. ACCURACY ~ POSITION_PRECISION.
-    // Note that POSITION_PRECISION isn't total error, it's convenient constant.
-    // True error can be up to several POSITION_PRECISION.
+    // To estimate acceptable error in a matrix coefficient, pick ACCURACY for this substep,
+    // set error in all other coefficients to zero and solve the system
+    // D(x_out) <= ACCURACY, D(y_out) <= ACCURACY for desired D(m_ij).
+    // Note that ACCURACY isn't equal to total error.
+    // Total error is larger than each ACCURACY, but still of the same magnitude.
+    // Via our choice of ACCURACY, we get a total error of up to several POSITION_PRECISION.
 
-    // Quantization steps (ACCURACY ~ POSITION_PRECISION):
+    // Quantization steps (pick ACCURARY = POSITION_PRECISION):
```

Comment:
Later you've used `pick:`, should be uniform. Also, maybe `use` or `set` here will be better?

Reply:
“Pick” conveys that the parameter is arbitrarily chosen to have this value. “Use” and “set” are more ambiguous. Frankly, I (still) don’t quite understand how `ACCURACY` is chosen or what it really is, and why that choice is OK, so “pick” would make me feel a little less lost.